<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>442</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>643</width>
    <height>440</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>643</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QChartView" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>-10</y>
     <width>571</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QTabWidget" name="Datapage">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>411</width>
     <height>441</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="Barpage">
    <attribute name="title">
     <string>条形图</string>
    </attribute>
    <widget class="QTableWidget" name="Bardata">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>401</width>
       <height>291</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="rowCount">
      <number>2</number>
     </property>
     <property name="columnCount">
      <number>5</number>
     </property>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>50</number>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>50</number>
     </attribute>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string>X轴类</string>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string>类1</string>
      </property>
     </item>
     <item row="0" column="2">
      <property name="text">
       <string>类2</string>
      </property>
     </item>
     <item row="0" column="3">
      <property name="text">
       <string>类3</string>
      </property>
     </item>
     <item row="0" column="4">
      <property name="text">
       <string>类4</string>
      </property>
     </item>
     <item row="1" column="0">
      <property name="text">
       <string>系列1:</string>
      </property>
     </item>
     <item row="1" column="1">
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item row="1" column="2">
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item row="1" column="3">
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item row="1" column="4">
      <property name="text">
       <string>0</string>
      </property>
     </item>
    </widget>
    <widget class="QWidget" name="widget_5" native="true">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>330</y>
       <width>281</width>
       <height>81</height>
      </rect>
     </property>
     <widget class="QPushButton" name="Save">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
     <widget class="QPushButton" name="DrawChart">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>绘制</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="large">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>0</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>^</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget_4" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>290</y>
       <width>401</width>
       <height>41</height>
      </rect>
     </property>
     <widget class="QLineEdit" name="Floatline">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>61</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>  浮点位:</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QSpinBox" name="Float">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>10</y>
        <width>44</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>4</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>2</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_6">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>10</y>
        <width>31</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>列:</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>31</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>行:</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QSpinBox" name="column">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>10</y>
        <width>44</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="value">
       <number>5</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="row">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>44</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>2</number>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="Piepage">
    <attribute name="title">
     <string>饼状图</string>
    </attribute>
    <widget class="QTableWidget" name="Piedata">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>401</width>
       <height>291</height>
      </rect>
     </property>
     <property name="rowCount">
      <number>3</number>
     </property>
     <property name="columnCount">
      <number>5</number>
     </property>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>50</number>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>50</number>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>37</number>
     </attribute>
     <row>
      <property name="text">
       <string>系列:</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>数据:</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>占比:</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>1</string>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
     </column>
     <column>
      <property name="text">
       <string>2</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>3</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>4</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>5</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string>类1</string>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string>类2</string>
      </property>
     </item>
     <item row="0" column="2">
      <property name="text">
       <string>类3</string>
      </property>
     </item>
     <item row="0" column="3">
      <property name="text">
       <string>类4</string>
      </property>
     </item>
     <item row="0" column="4">
      <property name="text">
       <string>类5</string>
      </property>
     </item>
     <item row="1" column="0">
      <property name="text">
       <string>30</string>
      </property>
     </item>
     <item row="1" column="1">
      <property name="text">
       <string>50</string>
      </property>
     </item>
     <item row="1" column="2">
      <property name="text">
       <string>60</string>
      </property>
     </item>
     <item row="1" column="3">
      <property name="text">
       <string>50</string>
      </property>
     </item>
     <item row="1" column="4">
      <property name="text">
       <string>10</string>
      </property>
     </item>
     <item row="2" column="0">
      <property name="text">
       <string>0.15</string>
      </property>
     </item>
     <item row="2" column="1">
      <property name="text">
       <string>0.25</string>
      </property>
     </item>
     <item row="2" column="2">
      <property name="text">
       <string>0.30</string>
      </property>
     </item>
     <item row="2" column="3">
      <property name="text">
       <string>0.25</string>
      </property>
     </item>
     <item row="2" column="4">
      <property name="text">
       <string>0.05</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QWidget" name="Linepage">
    <attribute name="title">
     <string>折线图</string>
    </attribute>
    <widget class="QTableWidget" name="Linedata">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>401</width>
       <height>291</height>
      </rect>
     </property>
     <property name="rowCount">
      <number>3</number>
     </property>
     <property name="columnCount">
      <number>5</number>
     </property>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>50</number>
     </attribute>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>50</number>
     </attribute>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
     <column>
      <property name="text">
       <string>1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>2</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>3</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>4</string>
      </property>
     </column>
     <item row="0" column="0">
      <property name="text">
       <string>系列1</string>
      </property>
     </item>
     <item row="0" column="1">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="0" column="2">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="0" column="3">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="0" column="4">
      <property name="text">
       <string/>
      </property>
     </item>
     <item row="1" column="0">
      <property name="text">
       <string>X</string>
      </property>
     </item>
     <item row="1" column="1">
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item row="1" column="2">
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item row="1" column="3">
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item row="1" column="4">
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item row="2" column="0">
      <property name="text">
       <string>Y</string>
      </property>
     </item>
     <item row="2" column="1">
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item row="2" column="2">
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item row="2" column="3">
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item row="2" column="4">
      <property name="text">
       <string>0</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="PieTool" native="true">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>140</y>
     <width>241</width>
     <height>152</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>尺寸</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="PieSize">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <double>0.100000000000000</double>
    </property>
    <property name="maximum">
     <double>1.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>0.500000000000000</double>
    </property>
   </widget>
   <widget class="QSlider" name="spin">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>360</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>旋转</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Divide">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>71</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>切片分离</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>0</y>
     <width>241</width>
     <height>161</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="Yup">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>90</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>刻度密度</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="BarCd">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>44</width>
      <height>25</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="BarYdown">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="BarYup">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>90</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Y轴规模：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Ydown">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_7">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>    ~</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>X轴规模：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Xdown">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Xup">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>51</width>
      <height>25</height>
     </rect>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string>    ~</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>  条宽</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="Barwidth">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>44</width>
      <height>25</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="value">
     <number>25</number>
    </property>
   </widget>
   <zorder>lineEdit_4</zorder>
   <zorder>lineEdit_7</zorder>
   <zorder>BarYdown</zorder>
   <zorder>label</zorder>
   <zorder>BarCd</zorder>
   <zorder>label_4</zorder>
   <zorder>BarYup</zorder>
   <zorder>Yup</zorder>
   <zorder>label_3</zorder>
   <zorder>Xup</zorder>
   <zorder>Xdown</zorder>
   <zorder>Ydown</zorder>
   <zorder>label_2</zorder>
   <zorder>Barwidth</zorder>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>290</y>
     <width>241</width>
     <height>151</height>
    </rect>
   </property>
   <widget class="QToolButton" name="colorB">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>颜色</string>
    </property>
   </widget>
   <widget class="QToolButton" name="fontB">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>字体</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>    标题：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="name">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <zorder>Datapage</zorder>
  <zorder>widget</zorder>
  <zorder>PieTool</zorder>
  <zorder>widget_2</zorder>
  <zorder>widget_3</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header>qchartview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
